src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -44,8 +44,8 @@ public void resetData_withValidReadOnlyAddressBook_replacesData() {
     @Test
     public void resetData_withDuplicateTags_throwsDuplicateTagException() {
         // Two persons with the same identity fields
-        Tag editedAlice = new TagBuilder(ALICE).build();
-        List<Tag> newPersons = Arrays.asList(ALICE, editedAlice);
+        Tag editedAlice = new TagBuilder(CS2103).build();
```

Comment:
Maybe the name of the variables here could be changed?

Reply:
Didn't catch that before. Thanks! I have changed it.